src/transaction/lock_manager.c

```diff
@@ -8518,18 +8518,37 @@ xlock_dump (THREAD_ENTRY * thread_p, FILE * outfp)
 
   /* compute number of lock res entries */
   num_locked = (int) lk_Gl.m_obj_hash_table.get_element_count ();
+  num_resource_alloc = (int) lk_Gl.m_obj_hash_table.get_alloc_element_count ();
+  num_entry_alloc = (int) lk_Gl.obj_free_entry_list.alloc_cnt;
+  size_alloc = (num_entry_alloc * sizeof (LK_ENTRY)) + (num_resource_alloc * sizeof (LK_RES));
 
   /* dump object lock table */
   fprintf (outfp, "Object Lock Table:\n");
   fprintf (outfp, "\tCurrent number of objects which are locked    = %d\n", num_locked);
-  fprintf (outfp, "\tMaximum number of objects which can be locked = %d\n\n", lk_Gl.max_obj_locks);
+  fprintf (outfp, "\tCurrent number of objects which are allocated = %d\n", num_resource_alloc);
```

Comment:
```suggestion
  fprintf (outfp, "\tCurrent number of objects which are allocated = %dBytes\n", num_resource_alloc);
```

Reply:
This is the number of locks allocated in memory.